mkdocs.yml

```diff
@@ -0,0 +1,87 @@
+# Configuration for all pages
+site_name: "MCAP"
+theme:
+  name: material
+  favicon: img/favicon32.png
+  logo: img/mcap.png
+  palette:
+    - scheme: default
+      toggle:
+        icon: material/brightness-7
+        name: Switch to dark mode
+      primary: "white"
+    - scheme: slate
+      toggle:
+        icon: material/brightness-4
+        name: Switch to light mode
+      primary: "black"
+  font:
+    text: Open Sans
+markdown_extensions:
+  - footnotes
+  - admonition
+  - pymdownx.highlight:
+      use_pygments: true
+  - pymdownx.superfences
+  - mdx_truly_sane_lists
+
+# links go directly to `.html` URL paths, this means we can preview the site in a GCS bucket.
+use_directory_urls: false
+
+# Site Layout
+nav:
+  - Home:
+      - "index.md"
+      - "home/messages-channels-and-schemas.md"
+      - "home/data-serialization-formats.md"
+  - Getting Started:
+      - "getting-started/overview.md"
+      - "getting-started/ros-1.md"
+      - "getting-started/ros-2.md"
+      - "getting-started/protobuf.md"
+      - "getting-started/json.md"
+      - "getting-started/visualize-in-foxglove-studio.md"
+  - Guides:
+      - Python:
+          - "guides/python/reading-writing-ros1.md"
+          - "guides/python/reading-writing-ros2.md"
+          - "guides/python/writing-jsonschema.md"
+          - "guides/python/reading-writing-protobuf.md"
+      - C++:
+          - "guides/cpp/writing-protobuf.md"
+  - API Reference:
+      - Python: docs/python/index.html
+      - C++: docs/cpp/index.html
+      - Typescript: docs/typescript/index.html
+      - Go: https://pkg.go.dev/github.com/foxglove/mcap/go/mcap
+      - Swift: docs/swift/documentation/mcap/
+
+  - More Resources:
+      - GitHub: "https://github.com/foxglove/mcap"
+      - mcap CLI: "https://github.com/foxglove/mcap/tree/main/go/cli/mcap"
+      - Slack: "https://foxglovedev.slack.com"
+      - Library Support: "support-matrix.md"
+      - ROS 2 Storage Plugin: "https://github.com/ros-tooling/rosbag2_storage_mcap"
+      - Specification:
+          - "specification/README.md"
+          - "specification/explanatory-notes.md"
+          - "specification/appendix.md"
```

Comment:
What do you all think about bringing the Specification section out towards the top level so it's not hidden away under "More Resources"?

Reply:
@amacneil  I hid this per our previous slack thread, do you have an opinion here?